spec/acceptance/reporting_spec.rb

```diff
@@ -55,54 +55,83 @@
     master.idempotent_apply(setup_manifest)
   end
 
-  context 'with report status: unchanged' do
-    context 'and noop_pending: false' do
-      let(:sitepp_content) do
-        # Puppet should report that nothing happens
-        ''
+  context 'with default incident_creation_report_attributes' do
+    context 'with report status: unchanged' do
```

Comment:
These (default tests) should be
```
context "report has 'failed_changes' attribute" do
  // incident creation test
end
context "report has 'corrective_changes' attribute" do
  // incident creation test
end
context "report does not have one of the default attributes" do
  context "but has 'intentional_changes' attribute" do
    // incident noop test
  end
  context "but has 'unchanged' attribute" do
    // incident noop test
  end
  context "but has 'pending_changes' attribute" do
    // incident noop test
  end
end
```

Reply:
This way, we also test that report with the non-default attributes _can_ noop on incident creation if the right conditions are met

Reply:
Resolving b/c these aren't PR blockers